### ISSUE TITLE
font error?

### ISSUE THREAD

**Issue by shagreobe** [fbdesignpro/sweetviz#23]
First off, love this! installed and run it on a 88K+ line dataset. loved how it came out.
Now, I got about 15 errors saying the same thing...
c:\users\shawn\python38\lib\site-packages\matplotlib\backends\backend_agg.py:214: RuntimeWarning: Glyph 37239 missing from current font.
  font.set_text(s, 0.0, flags=flags)
This might be my computer/matplotlib version (3.2.1) or who knows. 
thanks for a great lib

**Comment by fbdesignpro** [comment]
@shagreobe thank you for the great words and taking the time to report this! This looks like an unknown character (CJK unicode 37239?) in the provided font. Does that sound about right? I'm not sure how easy it is to get rid of that warning, but I will look into it!

**Comment by shagreobe** [comment]
I have since used several other datasets, and I have not got these errors again. May be something with the data.
Shawn

    On Wednesday, July 15, 2020, 8:46:02 AM EDT, Francois Bertrand <notifications@github.com> wrote:  
 
 


@shagreobe thank you for the great words and taking the time to report this! This looks like an unknown character (CJK unicode 37239?) in the provided font. Does that sound about right? I'm not sure how easy it is to get rid of that warning, but I will look into it!

**Comment by fbdesignpro** [comment]
Hello @shagreobe! It's been a while so you probably have moved on from that data, but I did want to let you know that version 1.1 which I just pushed live supports an INI override to specify a Chinese-Japanese-Korean (CJK) font which would help with the display and get rid of the warnings. (instructions are in the README)

If you ever get a chance to try it, do let me know if it works as it should!

Thanks again for your feedback,
Francois
